mobile_app_connector/models/compassion_correspondence.py

```diff
@@ -114,3 +115,70 @@ def _get_required_param(self, key, params):
         if key not in params:
             raise ValueError('Required parameter {}'.format(key))
         return params[key]
+
+    def mobile_get_preview(self, *_, **other_params):
+        body = self._get_required_param('letter-copy', other_params)
```

Comment:
A method comment explaining the function would be welcome. Please rename as well `_` parameter which interfers with the translation function of Odoo. `args` is more pythonic.